io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,33 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+        minimal:
+            threads: 5
+
+    blocksize: !mux
+        default: 
+            size: 1024
+     
+    runs: !mux
+        default: 
+            numruns: 2
+        minimal:
+            numruns: 1
+  
+    iteration: !mux
```

Comment:
Nope. The test is a test, it knows nothing about whether it's run as first or second or as the only one. It should simply perform the test. Then in results you see that this test was executed twice with following params and you can query for test files. I see you're storing a results, there is a `self.whiteboard` available for such operations, which is queryable from results so that would be actually a good place for it. Then you could ask for whiteboard of the first and the second test and compare them (you could still do it without the whiteboard, it'd just be more convenient this way as whiteboard is part of the json results and is queryable from database).